sdk/storage/azure-storage-file/src/test/java/com/azure/storage/file/spock/FileAsyncAPITests.groovy

```diff
@@ -27,8 +25,8 @@ class FileAsyncAPITests extends APISpec {
     FileAsyncClient primaryFileAsyncClient
     def shareName
     def filePath
-    static def defaultData = ByteBuffer.allocate(8).wrap("default".getBytes(StandardCharsets.UTF_8))
-    static def dataLength = defaultData.remaining()
+    def defaultData = ByteBuffer.allocate(8).wrap("default".getBytes(StandardCharsets.UTF_8))
```

Comment:
Shouldn't need to `allocate` first, just call `wrap`

Reply:
Done.